src/jdk.management.jfr/share/classes/jdk/management/jfr/RemoteRecordingStream.java

```diff
@@ -83,7 +82,7 @@
  *         rs.onEvent("jdk.GCPhasePause", System.out::println);
  *         rs.start();
  *     }
- * }
+ *     }
```

Comment:
Indentation?

Reply:
I want it to align with the starting brace.